cluster/saltbase/salt/calico/calico-node.manifest

```diff
@@ -0,0 +1,43 @@
+apiVersion: v1
```

Comment:
If you replaced this static pod with the equivalent daemonset today, what doesn't work?